Update CHANGELOG.md (#658)
bitwarden/sdk: crates/bws/CHANGELOG.md

## [Unreleased]

### Added

- Add a `BWS_CONFIG_FILE` environment variable to specify the location of the config file (#571)
- The `bws` CLI is now available as a Docker image (`docker run -it bitwarden/bws --help`) (#305)

> - `bws` mac signing support: (#535)

Reviewer: ```suggestion
- `bws` is code signed on Mac (#535)
```

> - The output option added back in the help menu: (#477)

Reviewer: This should probably be under `Fixed`.

```suggestion
- Re-add output options to the help menu after they were accidentally removed (#477)
```

Author: Moved this to a `Fixed` section.

> - We now use rustls on non-wasm platforms: (#374)

Reviewer: This is already under changed.
```suggestion
```

### Changed

> - Switched TLS backend to `rusttls`, removing the dependency on `OpenSSL`. (#374)

Reviewer: ```suggestion
- Switched TLS backend to `rusttls`, removing the dependency on `OpenSSL` (#374)
```

> - MSRV for `bws` was updated to `1.71.0`: (#589)

Reviewer: ```suggestion
- Updated MSRV for `bws` to `1.71.0` (#589)
```
